### DISCUSSION TITLE
IOHDIXController.kext ?

### DISCUSSION THREAD

**Discussion by cekkr** [PureDarwin/PureDarwin#148] [Q&A] [answered]
I would ask you about your requirement of IOHDIXController.kext ... do you need the source code? I guess that the binary itself without a "solid" license is useless for the project...

**Comment by csekel** [comment, marked as answer]
Sorry we will not use any binary Kext without available supporting source code and license

**Reply by cekkr** [reply]
Thank you for the clarification, I guess that would be obvious for the only people that can help you in this request. Unless you are also looking for a rewriting from scratch of the extension.
